app/mailers/audit_objects_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# Send report of invalid objects to admin team
+class AuditObjectsMailer < ActionMailer::Base
```

Comment:
An nice improvement is to create an application mailer so that all your mailers can inherited from? Instead of defining "layout mailer" and from address in every mailer